src/cryptoadvance/specter/server_endpoints/wallets_api.py

```diff
@@ -66,6 +67,22 @@ def generatemnemonic():
     }
 
 
+@wallets_endpoint_api.route("/parse_seedqr/", methods=["POST"])
+@login_required
+def parse_seedqr():
+    digitstream = request.form.get("digitstream")
+    bytestream = request.form.get("bytestream")
+
+    if digitstream:
+        mnemonic = seedqr.parse_standard_seedqr(digitstream)
+    else:
+        mnemonic = seedqr.parse_compact_seedqr(bytestream)
```

Comment:
`parse_compact_seedqr` already returns a string of the mnemonic - join should not be applied there.